### PR TITLE
feat: add version data compare in the history list

### LINKED ISSUE #7424: compare the history versioion differnt 【不同版本之间的配置比较查询】
**Is your feature request related to a problem? Please describe.**
if i want push configration to the product environment;  i should know both of the configration diffent ； 
测试环境的配置推送到线上时， 需要知道新增了哪些配置。 以便于了解新增记录；

**Describe the solution you'd like**
add compare options to the history version list;
在历史记录中添加比较功能， 比较当前版本和选定版本之间的配置差异。

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by MisterChangRay**
## What is the purpose of the change
fix #7424 

in the history list; add compare opration
在历史版本中添加比较选项， 比较版本和历史版本的差异。 方便了解历史配置变动

if  user want push configration to the product environment, the fuction can be used
这种需求可能会在用户上线时被使用， 通常来说 用户测试环境 需要迁移到线上环境时， 可以通过此功能分析配置变动

## Verifying this change
image
image

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

**Comment by realJackSun** [comment]
Looks Good.

File: console-ui/src/locales/en-US.js [hunk 1]
```diff
@@ -243,6 +243,10 @@ const I18N_CONF = {
     lastUpdateTime: 'Last Modified At',
     operator: 'Operator',
     operation: 'Operation',
+    compare: 'Compare',
+    historyCompareTitle: 'History Compare',
+    historyCompareLastVersion: 'Lasted Release Version',
```
Comment (ZebulonFeng) [inline console-ui/src/locales/en-US.js:248]: Maybe replace with these words are better?

historyCompareLastVersion -> historyCompareLatestVersion
Lasted  -> Latest

File: console-ui/src/pages/ConfigurationManagement/HistoryRollback/HistoryRollback.js [hunk 2]
```diff
@@ -195,6 +201,71 @@ class HistoryRollback extends React.Component {
     );
   }
 
+  goCompare(record) {
+    let tenant = getParams('namespace') || '';
+    let serverId = getParams('serverId') || 'center';
+    this.getConfig(-1, tenant, serverId, this.dataId, this.group).then(lasted => {
```
Comment (ZebulonFeng) [inline console-ui/src/pages/ConfigurationManagement/HistoryRollback/HistoryRollback.js:207]: lasted  -> latest

File: console-ui/src/pages/ConfigurationManagement/HistoryRollback/HistoryRollback.js [hunk 3]
```diff
@@ -195,6 +201,71 @@ class HistoryRollback extends React.Component {
...
+      this.getHistoryConfig(record.id, this.dataId, this.group).then(selected => {
+        this.diffEditorDialog.current.getInstance().openDialog(selected.content, lasted.content);
```
Comment (ZebulonFeng) [inline console-ui/src/pages/ConfigurationManagement/HistoryRollback/HistoryRollback.js:209]: lasted  -> latest

File: console-ui/src/locales/zh-CN.js [hunk 4]
```diff
@@ -242,6 +242,10 @@ const I18N_CONF = {
     lastUpdateTime: '最后更新时间',
     operator: '操作人',
     operation: '操作',
+    compare: '比较',
+    historyCompareTitle: '历史版本比较',
+    historyCompareLastVersion: '最新版本',
```
Comment (ZebulonFeng) [inline console-ui/src/locales/zh-CN.js:247]: historyCompareLastVersion -> historyCompareLatestVersion

File: console-ui/src/pages/ConfigurationManagement/HistoryRollback/HistoryRollback.js [hunk 5]
```diff
@@ -367,6 +438,12 @@ class HistoryRollback extends React.Component {
             />
             ,
           </div>
+          <DiffEditorDialog
+            ref={this.diffEditorDialog}
+            title={locale.historyCompareTitle}
+            currentArea={locale.historyCompareSelectedVersion}
+            originalArea={locale.historyCompareLastVersion}
```
Comment (ZebulonFeng) [inline console-ui/src/pages/ConfigurationManagement/HistoryRollback/HistoryRollback.js:445]: historyCompareLastVersion -> historyCompareLatestVersion

**Review by ZebulonFeng — CHANGES_REQUESTED** [review]
spell advice

**Pull request merged**